update-verifier/src/main.rs

```diff
@@ -25,25 +26,20 @@ fn clap_v3_styles() -> Styles {
         .placeholder(AnsiColor::Green.on_default())
 }
 
-fn main() {
-    if let Err(error) =
-        telemetry::start::<ExecContext, _>(LevelFilter::INFO, std::io::stdout)
-    {
-        error!(
-            ?error,
-            "update verifier encountered error while starting telemetry"
-        );
-        std::process::exit(1);
-    }
+fn main() -> eyre::Result<()> {
+    run().inspect_err(|error| error!(?error, "failed to run update-verifier"))
```

Comment:
Why do it this way?


Also, main should be using `color_eyre::Result` along with a `color_eyre::install()?`

Reply:
> Why do it this way?
No need to manually call process::exit like was being done befoe

> Also, main should be using color_eyre::Result along with a color_eyre::install()?
Sure